content/en/docs/deployment/releases.md

```diff
@@ -77,6 +77,30 @@ You can build specific releases using `rebar3 release -n <release_name>`
 
 ## Build Configuration
 
+### App Syntax {id="app-syntax"}
+
+```erlang
+{release, {myrelease, "0.1.0"},
+ [<app1>, <app2>]}
+```
+
+In a release, each `<app>` is an atom of the app name (e.g., `myapp`) or a tuple with the app name and optional extra elements:
+
+```erlang
+ {<app_name>,       % atom()
+  <app_vsn>,        % string()
+  <start_type>,     % permanent | transient | temporary | load | none
+  <included_apps>}  % [atom()]
+```
+
+These elements correspond to items in the [release resource file](//erlang.org/doc/man/rel.html) and are further documented there.
+
+- Elements after `<app_name>` may be omitted, as long as the order is preserved.
+- `<app_vsn>` is commonly omitted, in which case it is determined from `.app.src`. Otherwise, `<app_vsn>` must match `.app.src`.
+- If present, `<start_type>` and `<included_apps>` are written as-is to the `.rel` file, and follow the same defaulting logic:
+    - `<start_type>` defaults to `permanent`.
+    - `<included_apps>` defaults to the apps from `.app.src`.
+
```

Comment:
Do you think it'd make sense to have a literal example like "here's what an actual release config would look like" ? I'm personally fine without it, but I know some readers deal better with discrete examples without placeholders (or foo/bar examples)

Reply:
Good idea! Here's a v2 which includes it